rewrite-javascript/rewrite/src/javascript/format.ts

```diff
@@ -704,6 +704,13 @@ export class MinimumViableSpacingVisitor extends JavaScriptVisitor<ExecutionCont
         });
     }
 
+    protected async visitThrow(thrown: J.Throw, p: ExecutionContext): Promise<J | undefined> {
+        const ret = await super.visitThrow(thrown, p) as J.Throw;
+        return ret && produce(ret, draft => {
+           draft.exception.prefix.whitespace = " ";
```

Comment:
We should take care to only update the `Space` when required. This is basically only when `whitespace` is empty and there are no comments. This should apply to all modifications performed by this visitor.